### PR TITLE
Ping: Add -s argument to specify the payload size of the ping

### PR DESCRIPTION
**Pull request by twvd**

### CODE REVIEW

File: Userland/Libraries/LibC/netinet/ip.h [hunk 1]
```diff
@@ -6,4 +6,32 @@
...
 
+#include <assert.h>
+#include <bits/stdint.h>
+#include <sys/cdefs.h>
+
 #include "in.h"
+
+__BEGIN_DECLS
+
+struct [[gnu::packed]] ip {
```
Comment (alimpfard) [inline Userland/Libraries/LibC/netinet/ip.h:17]: This is a C header, please use `__attribute__((packed))`.

File: Userland/Libraries/LibC/netinet/ip.h [hunk 2]
```diff
@@ -6,4 +6,32 @@
...
+    uint16_t ip_len;
+    uint16_t ip_id;
+    uint16_t ip_off;
+    uint8_t ip_ttl;
+    uint8_t ip_p;
+    uint16_t ip_sum;
+    struct in_addr ip_src;
+    struct in_addr ip_dst;
+};
+static_assert(sizeof(struct ip) == 20, "struct ip: invalid length");
```
Comment (alimpfard) [inline Userland/Libraries/LibC/netinet/ip.h:35]: Same C header comment, you could try `_Static_Assert()` or whatever that was, or just hide it in an ifdef

Comment (twvd) [reply Userland/Libraries/LibC/netinet/ip.h:35]: There is a C definition for static_assert here, which I used:

https://github.com/SerenityOS/serenity/blob/dc9f516339be8850bbc0b4a1cc2051a72bf9b07a/Userland/Libraries/LibC/assert.h#L30

Happy to use _Static_assert if you want still, or is this ok?

Comment (alimpfard) [reply Userland/Libraries/LibC/netinet/ip.h:35]: Ah, no worries then.
`static_assert` looks better, and works with C++ too~

**Comment by twvd** [comment]
Thanks again @alimpfard , I addressed the comment.

**Pull request merged**